v4.5.0 does not pass it's own `build` task (fails on flake8)

I have checked out source for 4.5.0 and tried 

```$ pypyr ops/build
--> loading repo pypyr-config.yaml
--> check expected keys in pypyr-config.yaml
--> verify setup.py metadata
running check
--> flake8 linting
./setup.py:119:10: E126 continuation line over-indented for hanging indent
./setup.py:120:6: E121 continuation line under-indented for hanging indent
./pypyr/retries.py:63:36: E226 missing whitespace around arithmetic operator
./pypyr/retries.py:111:26: E226 missing whitespace around arithmetic operator
./pypyr/utils/filesystem.py:460:9: W503 line break before binary operator
./pypyr/utils/filesystem.py:461:9: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:107:13: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:114:13: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:121:13: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:128:13: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:136:28: W504 line break after binary operator
./tests/unit/pypyr/formatting_test.py:146:28: W504 line break after binary operator
./tests/unit/pypyr/formatting_test.py:603:13: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:625:13: W503 line break before binary operator
./tests/unit/pypyr/formatting_test.py:641:13: W503 line break before binary operator
./tests/unit/pypyr/retries_test.py:495:38: W504 line break after binary operator
./tests/unit/pypyr/steps/pype_test.py:401:37: W504 line break after binary operator
./tests/unit/pypyr/steps/pype_test.py:711:37: W504 line break after binary operator
./tests/unit/pypyr/steps/pype_test.py:794:37: W504 line break after binary operator
./tests/integration/pypyr/utils/filesystem_int_test.py:75:21: W503 line break before binary operator
./tests/integration/pypyr/utils/filesystem_int_test.py:95:21: W503 line break before binary operator
Error while running step pypyr.steps.cmd at pipeline yaml line: 28, col: 5
Error while running step pypyr.steps.call at pipeline yaml line: 7, col: 5
Something went wrong. Will now try to run on_failure.

CalledProcessError: Command '['flake8']' returned non-zero exit status 1.
```

--- a/pypyr/retries.py
+++ b/pypyr/retries.py
@@ -60,7 +60,7 @@ class BackoffBase(abc.ABC):
         It allows you to jitter under or over the sleep interval depending on
         if jrc > 1.
         """
-        return random.uniform(sleep*self.jrc, sleep)
+        return random.uniform(sleep * self.jrc, sleep)
 
 # endregion backoff base
 
@@ -108,7 +108,7 @@ class linear(BackoffBase):
 
     def __call__(self, n):
         """Return the product of n * sleep."""
-        return self.min(n*self.sleep)
+        return self.min(n * self.sleep)
 
 
 class linearjitter(linear):

--- a/setup.cfg
+++ b/setup.cfg
@@ -17,6 +17,7 @@ fail_under = 100
 branch = True
 
 [flake8]
+ignore = W503
 exclude = .git,__pycache__,.tox,*.egg,build,data,dist,.cache,.env
 
 [metadata]

--- a/setup.py
+++ b/setup.py
@@ -116,8 +116,8 @@ setup(
     # installed, specify them here.  If using Python 2.6 or less, then these
     # have to be included in MANIFEST.in as well.
     package_data={
-         'pypyr': ['pipelines/*'],
-     },
+        'pypyr': ['pipelines/*']
+    },
 
     # Although 'package_data' is the preferred approach, in some case you may
     # need to place data files outside of your packages. See:

--- a/tests/unit/pypyr/formatting_test.py
+++ b/tests/unit/pypyr/formatting_test.py
@@ -133,9 +133,9 @@ def test_recursive_formatter_manual_to_auto():
     with pytest.raises(ValueError) as err:
         RecursiveFormatter().format('{0} {}', 'a', 'b')
 
-    assert (str(err.value) ==
-            ('cannot switch from manual field specification to automatic '
-             'field numbering'))
+    assert (str(err.value)
+            == ('cannot switch from manual field specification to automatic '
+                'field numbering'))
 
 
 def test_recursive_formatter_auto_to_manual():
@@ -143,9 +143,9 @@ def test_recursive_formatter_auto_to_manual():
     with pytest.raises(ValueError) as err:
         RecursiveFormatter().format('{} {1}', 'a', 'b')
 
-    assert (str(err.value) ==
-            ('cannot switch from manual field specification to automatic '
-             'field numbering'))
+    assert (str(err.value)
+            == ('cannot switch from manual field specification to automatic '
+                'field numbering'))
 
 # endregion RecursiveFormatter.format
 

--- a/tests/unit/pypyr/retries_test.py
+++ b/tests/unit/pypyr/retries_test.py
@@ -492,8 +492,8 @@ def test_backoff_export_list():
     """
     members = inspect.getmembers(
         pypyr.retries,
-        lambda m: inspect.isclass(m) and
-        issubclass(m, pypyr.retries.BackoffBase))
+        lambda m: inspect.isclass(m)
+        and issubclass(m, pypyr.retries.BackoffBase))
 
     assert pypyr.retries.builtin_backoffs == {
         n[0]: n[1] for n in members if n[0] != 'BackoffBase'}

--- a/tests/unit/pypyr/steps/cmd_step_test.py
+++ b/tests/unit/pypyr/steps/cmd_step_test.py
@@ -8,6 +8,14 @@ import pytest
 import subprocess
 
 
+@pytest.fixture(autouse=True)
+def lang_c(monkeypatch):
+    """Ensure commands print messages in English
+    Without this, running tests in non-english locale print unexpected messages
+    """
+    monkeypatch.setenv("LANG", "C")
+
+
 def test_cmd_single_word():
     """One word command works."""
     context = Context({'cmd': 'date'})

--- a/tests/unit/pypyr/steps/cmd_step_test.py
+++ b/tests/unit/pypyr/steps/cmd_step_test.py
@@ -10,7 +10,8 @@ import subprocess
 
 @pytest.fixture(autouse=True)
 def lang_c(monkeypatch):
-    """Ensure commands print messages in English
+    """Ensure commands print messages in English.
+
     Without this, running tests in non-english locale print unexpected messages
     """
     monkeypatch.setenv("LANG", "C")

--- a/tests/unit/pypyr/steps/pype_test.py
+++ b/tests/unit/pypyr/steps/pype_test.py
@@ -398,8 +398,8 @@ def test_pype_get_args_and_pipearg():
 
 def mocked_run_pipeline(*args, **kwargs):
     """Check pipeline name set on context in child pipeline."""
-    assert (kwargs['pipeline_name'] ==
-            kwargs['context'].pipeline_name == 'pipe name')
+    assert (kwargs['pipeline_name']
+            == kwargs['context'].pipeline_name == 'pipe name')
 
 
 @patch('pypyr.pipelinerunner.load_and_run_pipeline')
@@ -708,8 +708,8 @@ def test_pype_no_pipe_arg(mock_run_pipeline):
 
 def mocked_run_pipeline_with_runtime_error(*args, **kwargs):
     """Check pipeline name set on context in child pipeline with arb err."""
-    assert (kwargs['pipeline_name'] ==
-            kwargs['context'].pipeline_name == 'pipe name')
+    assert (kwargs['pipeline_name']
+            == kwargs['context'].pipeline_name == 'pipe name')
     raise RuntimeError('whoops')
 
 
@@ -791,8 +791,8 @@ def test_pype_use_parent_context_with_swallow(mock_run_pipeline):
 
 def mocked_run_pipeline_with_stop(*args, **kwargs):
     """Check pipeline name set on context in child pipeline with Stop."""
-    assert (kwargs['pipeline_name'] ==
-            kwargs['context'].pipeline_name == 'pipe name')
+    assert (kwargs['pipeline_name']
+            == kwargs['context'].pipeline_name == 'pipe name')
     raise Stop()
 
 

